AzureCalling/Controllers/ParticipantListViewController.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class ParticipantListViewController: UIViewController, BottomDrawerViewController {
+
+    // MARK: Properties
+
+    private var participantList: [BottomDrawerItem] = [BottomDrawerItem]()
+    var participantTable: UITableView!
+    var callingContext: CallingContext!
```

Comment:
I don't think we need to inject the whole callingContext into this class. We just need the remote participant list

Reply:
We also need our own name and mute status.  Once we implement the live participant list, this will expand to an observer to our own mute status as well and possibly our own speaking indicator.

Reply:
Defined a struct `ParticipantListInfo` to carry the necessary info to build the participant list, no longer rely on the entire `callingContext`